scripts/sysman/EXT_EXP_MemState.rst

```diff
@@ -0,0 +1,78 @@
+<%
+import re
+from templates import helper as th
+%><%
+    OneApi=tags['$OneApi']
+    x=tags['$x']
+    X=x.upper()
+    s=tags['$s']
+    S=s.upper()
+%>
+:orphan:
+
+.. _ZES_extension_memory_state:
```

Comment:
According to the SAS 
"An Intel SW agent (XPUM/L0/KMD?) running in the VM will collect graphics memory consumption statistics from the OS video memory manager. This agent will 'push' the data down the stack and to the PF KMD via the VF2PF relay mechanism. "

According to https://docs.intel.com/documents/GfxSWAT/Common/GFX_SRIOV_SAS/ATS-M_PVC/Graphics%20SRIOV%20SAS/GraphicsSRIOVSAS.html#pf-kmd-sriov-startup-runtime , Telemetry From in-VM GPU SW Sources section , we need a Sysman API to push the memory utilization data down to the Guc

Please share whether this API will be covered in this Spec.


Thanks

Reply:
@joshuaranjan closed on this offline with @sanilkumar0 . looks like we don't need API for that